app/views/accounts/two_factor_authentication/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1 class="mt0">
```

Comment:
I've noticed that a few of these new pages lack a distinct page title describing its purpose ("Two-factor authentication", "History", maybe others).

Related: https://www.w3.org/WAI/WCAG21/Understanding/page-titled.html

Should be relatively straight-forward to include here, using the same labels as in the navigation:

```suggestion
<% title t('account.navigation.two_factor_authentication') %>

<h1 class="mt0">
```

Reply:
For context, I'd stumbled on this in discovering some other, preexisting issues with page titles.

See LG-3502 for full details.

Reply:
good catch, thank you. I added a title to all of the new account pages in https://github.com/18F/identity-idp/pull/4209/commits/5006dce6e3dcfbf9667de6c204ce5c2f7b74c1c5